src/app/user.ts

```diff
@@ -0,0 +1,6 @@
+import {oauth} from '../store';
```

Comment:
maybe rename this to smth. like oauthStore , "oauth" can be anything.

Reply:
Fair enough. I initially favored `oauth` because it is shorted and imported from `../store`, but wasn't sure.  I'll make the change since it has already been a source of confusion in the first PR :)

Reply:
I'll also change `config` => `configStore`